lua/lsp/ruby-ls.lua

```diff
@@ -0,0 +1,5 @@
+-- If you are using rvm, make sure to change below configuration
+require'lspconfig'.solargraph.setup {
+    cmd = { DATA_PATH .. "~/.rbenv/shims/solargraph", "--stdio" },
+    on_attach = require'lsp'.common_on_attach
+}
```

Comment:
Consider adding the handler, you can find it in the python lsp module and then pass the ruby diagnostic values here